store/sql_channel_store.go

```diff
@@ -52,7 +52,7 @@ func ClearChannelCaches() {
 	channelByNameCache.Purge()
 }
 
-func NewSqlChannelStore(sqlStore *SqlStore) ChannelStore {
+func NewSqlChannelStore(sqlStore SqlStore) ChannelStore {
```

Comment:
Why isn't this a pointer?  isn't it copying it now on every new storage creation?